.gitignore

```diff
@@ -37,7 +37,9 @@ cl/settings/*private.py
 cl/settings/google_auth.json
 .DS_Store
 .dir-locals.el
-
-cl/node_modules
 /.env.dev
 /.env
+
+.vscode/*
+docker/scripts
```

Comment:
These are some custom scripts to perform some basic Docker tasks (e.g. update images for everything) that C. Johnson shared with me.

Reply:
Sorry, Samier, I know we're struggling to get this one out the door, but I'm doing one more review and I think we should remove this line. Either:

1. These scripts should be shared with everybody, and therefore this line shouldn't be here, or...
2. These scripts are only useful to you, in which case it doesn't merit a line in the `.gitignore`. 

I hope you're not frustrated with all my nitpicking. I debated mentioning this at all, but I think it's important to do so because I hate shadow-scripts. If there's a script that C. Johnson wrote that's useful, we should get it merged, not build an exception for it, right?